src/main/java/flashcard/types/FlashcardList.java

```diff
@@ -0,0 +1,196 @@
+package flashcard.types;
+
+import exceptions.EmptyListException;
+import exceptions.FlashCLIillegalArgumentException;
+
+import static constants.ErrorMessages.CREATE_INVALID_ORDER;
+import static constants.ErrorMessages.CREATE_MISSING_FIELD;
+import static constants.ErrorMessages.CREATE_MISSING_DESCRIPTION;
+import static constants.ErrorMessages.VIEW_OUT_OF_BOUNDS;
+import static constants.ErrorMessages.EMPTY_LIST;
+import static constants.SuccessMessages.CREATE_SUCCESS;
+import static constants.SuccessMessages.DELETE_SUCCESS;
+import static constants.SuccessMessages.VIEW_ANSWER_SUCCESS;
+import static constants.SuccessMessages.VIEW_QUESTION_SUCCESS;
+import static constants.SuccessMessages.EDIT_SUCCESS;
+import static constants.SuccessMessages.LIST_SUCCESS;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+public class FlashcardList {
+    public static ArrayList<Flashcard> flashcards = new ArrayList<>();
+
+    /**
+     * Creates a new flashcard
+     *
+     * <p>The arguments must contain both a question (denoted by "/q") and an answer (denoted by "/a").
+     * The question must appear before the answer in the input string.</p>
+     *
+     * @param arguments A string with the flashcard details
+     * @return A success message indicating the flashcard has been created.
+     * @throws FlashCLIillegalArgumentException If required fields are missing,
+     *         the question and answer are in the wrong order, or either field is empty.
+     */
+    private static final Logger logger = Logger.getLogger(FlashcardList.class.getName());
+
+    public static String createFlashcard(String arguments) throws FlashCLIillegalArgumentException {
```

Comment:
This function is a little long and difficult to read. Consider refactoring it into multiple fns.